### PR TITLE
feat: opt-in sentry.io error reporting

### PR DESCRIPTION
**Pull request by Benjozork**
## Summary of Changes

This adds **opt-in**, **anonymous** error reporting via sentry.io.

image

* [x] Add sentry.io integration client
* [x] Add consent popup
* [x] Add EFB toggle
* [x] Add initialization code for leagcy JS gauges
* [x] Add initialization code for TS gauges

This is enabled on all production builds. This will not run on local or PR builds.

Clients are instantiated on all instruments, and a "root" client on the EFB performs the consent gathering flow.

## Testing instructions

* Make sure you get the consent popup on loading the aircraft and that you do not subsequently get it again
* Make sure the EFB toggle works
* Contact a dev team member to test if the consent is being handled properly

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: src/sentry-client/src/FbwAircraftSentryClient.ts [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+            popup.showPopUp(
+                'A32NX - ERROR REPORTING',
+                'Are you willing to help us by enabling anonymous error reporting? This is 100% optional and we will never collect your personal data, '
+                + 'but it will help us diagnose issues quickly.',
+                'normal',
+                () => resolve(true),
+                () => resolve(false),
+            );
+        });
+    }
```
Comment (ErickSharp) [inline src/sentry-client/src/FbwAircraftSentryClient.ts:140]: Can you clarify to the user what you want them to do by asking them a question? Currently they just see a Yes and No button Yes what? No what?

Comment (ErickSharp) [reply src/sentry-client/src/FbwAircraftSentryClient.ts:140]: Sorry was looking at the old message

**Review by wpine215 — APPROVED** [review]

File: src/fmgc/src/components/fms-messages/FmsMessages.ts [hunk 2]
```diff
@@ -1,3 +1,8 @@
+// Copyright (c) 2021-2022 FlyByWire Simulations
+// Copyright (c) 2021-2022 Synaptic Simulations
```
Comment (davidwalschots) [inline src/fmgc/src/components/fms-messages/FmsMessages.ts:2]:
```suggestion
```

File: src/fmgc/src/efis/EfisSymbols.ts [hunk 3]
```diff
@@ -1,5 +1,7 @@
-//  Copyright (c) 2021 FlyByWire Simulations
-//  SPDX-License-Identifier: GPL-3.0
+// Copyright (c) 2021-2022 FlyByWire Simulations
+// Copyright (c) 2021-2022 Synaptic Simulations
```
Comment (davidwalschots) [inline src/fmgc/src/efis/EfisSymbols.ts:2]:
```suggestion
```

**Review by davidwalschots — CHANGES_REQUESTED** [review]

File: src/fmgc/src/efis/EfisSymbols.ts (hunk 3 above)
Comment (Benjozork) [reply src/fmgc/src/efis/EfisSymbols.ts:2]: This is in the FMGC, therefore intended.

**Pull request merged**